src/wp-admin/update-core.php

```diff
@@ -1011,11 +1011,30 @@ function do_undismiss_core_update() {
 	$help_sidebar_autoupdates = '<p>' . __( '<a href="https://wordpress.org/documentation/article/plugins-themes-auto-updates/">Documentation on Auto-updates</a>' ) . '</p>';
 }
 
+$help_sidebar_rollback = '';
+
+if ( current_user_can( 'update_themes' ) || current_user_can( 'update_plugins' ) ) {
+	$rollback_help = '<p>' . __( 'Rollback is a feature that will create a temporary backup of a plugin or theme before it is upgraded. This backup is used to restore the plugin or theme back to its previous state if there is an error during the update process.' ) . '</p>';
```

Comment:
New first sentence:
`This feature will create a temporary backup of a plugin or theme before it is upgraded.`